_posts/2018-07-12-the-case-for-open-source-software.md

```diff
@@ -0,0 +1,176 @@
+[GitHub](http://github.com/llnl) increased exposure of open source
+code. We launched a [software portal](http://software.llnl.gov) in
+2015 to provide additional access and track overall trends. Thanks to
+growing institutional support over the last 10 years, our involvement
+with the open source community has taken off.
+
+<figure>
+  <img src="{{site.baseurl}}/assets/blog/llnl/llnl-repo-graph.png" alt="Graph showing LLNL’s open source repository creation
+  history with years marked on the x-axis and the cumulative number of
+  repositories on the y-axis."/>
+</figure>
+
+Our software addresses just about every key computing need: compilers,
+package managers, data analysis tools, visualization tools, input/output
+benchmarking, data storage, parallel performance, workload management,
+math and physics codes, and more. We work in multiple operating systems
+and programming languages.
+
+# Win–wins
+
+As 18F [points
+out](https://18f.gsa.gov/search/?q=open+source+software), adopting an
+open source policy has many benefits. Government agencies — especially
+those with HPC centers, like LLNL — are trying to solve similar
+problems. With open source software, agencies and contractors can align
+their goals and make the most of external resources. We can share
+projects and avoid redundancies. (After all, not everyone can build and
+support their own football field–sized supercomputing facilities.)
+
+<div class="testimonial-blockquote">
+  With open source software [OSS], we are in control and can develop quickly and efficiently. It gives us the freedom to choose the best
+  computer hardware instead of being locked into one company’s hardware
+  solution. OSS is not for every organization, but for an organization
+  like LLNL, whose job is to build innovative solutions for national
+  security, OSS has been an essential tool.
+    <span>- Terri Quinn, LLNL’s Advanced Simulation and Computing Program leader</span>
+</div>
+
+For example, we’re developing open source tools with the NNSA’s
+[Advanced Simulation and Computing Program](https://www.exascaleproject.org/more-on-the-software-that-underpins-the-exascale-computing-project/) to make HPC codes portable to next-generation architectures. The DOE’s [Exascale Computing Project](https://www.exascaleproject.org/) depends on open source technology to share work among participating institutions, including LLNL and other national labs.
+
+One of our most popular open source HPC projects is ZFS on Linux, which,
+as the name states, is a version of the ZFS file system that supports
+the Linux operating system. Embodying the community’s quid pro quo, we
+could adapt the software to Linux because the [original
+ZFS](http://open-zfs.org/wiki/History) (which only supported the
+Solaris operating system) was open source.
+
+We create software that resonates with the open source community due to
+its flexibility with a variety of use cases. LNLL’s Spack, a collection
```

Comment:
Minor typo: `LNLL` -> `LLNL`

Reply:
Also, can we add a link of `[LLNL's Spack](https://spack.io)`

Reply:
Typo fixed and link added